src/m1_facilitator/SeedDataInitializer.ts

```diff
@@ -0,0 +1,197 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Web3 from 'web3';
+import * as Mosaic from 'Mosaic';
+import BigNumber from 'bignumber.js';
+
+import Gateway, { GatewayType } from './models/Gateway';
+import Anchor from './models/Anchor';
+import Repositories from './repositories/Repositories';
+import ContractEntity, { EntityType } from '../common/models/ContractEntity';
+import Utils from '../common/Utils';
+
+export default class SeedDataInitializer {
+  /** Instance of Repositories class. */
+  private repositories: Repositories;
+
+  /**
+   * Construct the SeedDataInitializer with the params.
+   *
+   * @param repositories Instance of repository class.
+   */
+  public constructor(
+    repositories: Repositories,
+  ) {
+    this.repositories = repositories;
+  }
+
+  /**
+   * Generate and save records that should be populated as seed data.
```

Comment:
Let's have more verbose documentation wrt what exactly it's saving (in details).

Reply:
Currently documentation already explains what entity it's saving. Can you elaborate what else can be added?